### ISSUE TITLE
Weird "Ball of Light" around vehicle in one area of the map.

### ISSUE THREAD

**Issue by Dylan-M** [MapsOfNorway/rennebugs#115]
Hard to describe, so I've attached a bunch of screenshots. Only happens when I activate the spreader.

![fsScreen_2022_08_11_20_32_17](https://user-images.githubusercontent.com/545408/185814293-19f00616-4a26-4a50-b0f5-7d864ef8ec1f.png)
![fsScreen_2022_08_11_20_32_24](https://user-images.githubusercontent.com/545408/185814295-f5b4de65-ad6f-4b58-b4d0-cca8c8cddd34.png)
![fsScreen_2022_08_11_20_32_29](https://user-images.githubusercontent.com/545408/185814298-e13498f5-78ab-42e1-a06f-68f235e6be0d.png)
![fsScreen_2022_08_11_20_32_33](https://user-images.githubusercontent.com/545408/185814300-1994b232-4cc7-49aa-9c92-899bf9a69d3a.png)
![fsScreen_2022_08_11_20_32_57](https://user-images.githubusercontent.com/545408/185814303-9f1ab7c0-5692-43fc-b7b3-901c0aa14f2c.png)
![fsScreen_2022_08_11_20_32_59](https://user-images.githubusercontent.com/545408/185814305-5a91f583-fc95-42b0-9a86-fbfe5599a1e2.png)
![fsScreen_2022_08_11_20_33_01](https://user-images.githubusercontent.com/545408/185814309-38a653b3-f8d5-46a6-a714-5a1830ec794e.png)
![fsScreen_2022_08_11_20_33_07](https://user-images.githubusercontent.com/545408/185814311-0a7d4650-b7e5-450a-9a6c-e482c0147c57.png)

**Comment by OlaHaldor** [comment]
Is it in specific parts of the map or anywhere?
This must be new. 

Is it with the 1.7 patch?
Might be more of a Giants engine thing than the map itself.

**Comment by Dylan-M** [comment]
It seems to happen at random all over the map. This is the only map with the issue, hence why I reported it as a map issue rather than an engine issue.

**Comment by OlaHaldor** [comment]
The other maps you've tested, are they large like Rennebu too?
Just trying to pin point whether it could be a render error because of size or shaders or.. I don't know.

**Comment by Dylan-M** [comment]
Rennebu is larger. All my other maps are 1x and 4x. So yes, perhaps that has something to do with it. If I can squeeze out time in the near future, I'll test it on another 16x map.

**Comment by OlaHaldor** [comment]
I have tested this now on Rennebu and I don't see this issue at all. Don't know what to make of it.
Does the log produce any error or warning ?

**Comment by Dylan-M** [comment]
I think I figured it out. I think it is an interesting combination of 3 things:
1. 16x Map
2. All the "Real Dirt" mods
3. Teleporting very long distances using Easy Dev

We can probably close this as nothing to do with the map, though I do not experience it on the Wyoming map with the same setup.

I did find a workaround too: Exit out of the Easy Dev menu as soon as you click the teleport, and before the vehicle stops flying across the map. In that event it doesn't happen.
